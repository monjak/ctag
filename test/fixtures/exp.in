1 [MBS] Green Sun's Zenith
1 [SOM] Corrupted Harvester
