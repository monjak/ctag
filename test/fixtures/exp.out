1x [card=e:mbs Green Sun's Zenith][MBS] Green Sun's Zenith[/card]
1x [card=e:som Corrupted Harvester][SOM] Corrupted Harvester[/card]
